docs/pages/guides/react-native-integration.mdx

```diff
@@ -106,36 +127,114 @@ useEffect(() => {
 ```
 
 ## Usage
+
+Mobile Wallet Protocol Client provides 2 interfaces for mobile app to interact with the Smart Wallet, an EIP-1193 compliant provider interface and a wagmi connector.
+
+<Callout type="success" >
+If your app is using wallet aggregator, go straight to **Option 2: Wagmi Connector** for 1-line integration. :)
```

Comment:
Nit: Might be nice to link to the Option 2 fragment here: `#option-2-wagmi-connector`